components/dialog/src/vwc-dialog.scss

```diff
@@ -71,3 +71,26 @@
 		width: 100%;
 	}
 }
+
+.mdc-dialog {
+	$top-position: --_top-position;
```

Comment:
common practice is to map these in a separate variables partial file. if only used in the file it's fine

Reply:
This is the only use. Thought it didn't worth opening a file for it